### PR TITLE
Open Door Policy

### PR DESCRIPTION
**Pull request by Pebblesthe2nd**
## About The Pull Request

Fixed the hard_open var, now the animations show up for all doors
(Actually didn't FIX it, just made 1 to FALSE, so yeah.)

Overlapping door systems cleaned up.
New door type secure = ID locked, for places like armories, can't be padlocked. New sprite.
New door type cell = same as above
clean house, dirty room just weathering layers added to existing door variants for more options.

Removed unpowered/shuttle, a remnant
Repathed the duplicate unpowered wood door, replaced with secure door if ID locked, simple wooden door if not. (reason both used just due to messy assets confusing mappers)

Iron door slightly faster opening. but separated from its parent metal door, which now is called a simple metal door. No mechanical difference.

Adjusts some doorframes, animations, visual stuff, but pretty subtle.

Also tweaks the inhand sheet construction a little, instead of store door you make a simple metal door using 10 metal now (where was the glass? Nowhere, thats where.)

Since editing inhand stack craft anyways, small bonus:
Comments out the following items from the inhand list thats really, really redundant to keep in stack crafting menu. APC frame (not supposed to build, easy as pie to re enable if it ever gets fixed.) SM reflector, Air alarm frame, about a third of the base SS 13 airlocks, plenty of the placeholders remain but less clutter.

Restructure/deleting duplicates of relevant stuff. Also murdering the icon diff bot by doing this, apparantly.

TL:dr : Mapping assets mostly, adds documentation, couple visual/practical changes to some doors. A few metal doors using iron door sprite now got their own to separate. 

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
add: New door types, sorts out old ones, edits/makes some door sprites/animations
del: Instead of doors spread out in 3 dmis, all in one now.
tweak: comments out couple redundant items from metal stack craft list for QoL
/:cl:

### CODE REVIEW

**Comment by buffyuwu** [comment]
rerunning diffs job

**Comment by buffyuwu** [comment]
sigh diffsbot is having issues again

**Comment by AffectedArc07** [comment]
> sigh diffsbot is having issues again

Its having issues because you used scuffed names in your icon states
image

**Comment by buffyuwu** [comment]
@Pebblesthe2nd 

the exception was a real error

**Comment by Pebblesthe2nd** [comment]
> Its having issues because you used scuffed names in your icon states > 

Thanks for the tip, but it wasn't it. Renamed then deleted the divider, still same error for the bot. Maybe it's malf. Gonna peek around in the dmis a bit and see if anything stands out.

**Comment by AffectedArc07** [comment]
It's definitely a state issue and not a recurring issue with the bot, as other DMIs render.

State that's extremely long maybe?

**Comment by AffectedArc07** [comment]
There you go
image

**Comment by buffyuwu** [comment]
ate icons
ate icon states
ate diffschecker
ate 76 files changed


simple as

**Review by buffyuwu — CHANGES_REQUESTED** [review]
pls fix conflicts ily

**Review by buffyuwu — APPROVED** [review]
